pandas/tests/tseries/test_offsets.py

```diff
@@ -41,6 +41,8 @@
 from pandas.tseries.holiday import USFederalHolidayCalendar
 
 
+data_dir = tm.get_data_path()
```

Comment:
Moving this call to up here ensures that we get the same `data_dir` whether running the tests via pytest or interactively.  Under the status quo, copy/pasting the pertinent test below will fail because `get_data_path` will not behave as expected.

Reply:
huh? we use this pattern everywhere, why are you changing this?

Reply:
Because when I try to run these tests interactively and copy/paste the contents of a test function, `tm.get_data_path` returns unexpected results depending on `os.getcwd()`.  AFAICT when run non-interactively it behaves as if cwd is pandas/tests/tseries.

Reply:
what do you mean 'interactively'? you should simply be running

``pytest pandas/tests/...... -k ...`` or whatever that is the idiomatic way to run tests.

Reply:
When a test fails and I want to figure out why, I run the contents of the test manually in the REPL.

Reply:
Happy to revert this change; not that big a deal.

Reply:
yes pls revert.

standard way to run tests is
```
pytest path/to/test -k optional_regex
```
lots of options, including ``--pdb`` to drop into the debuger

pls revert this is non-standard